src/components/RangeSlider/components/DualThumb/DualThumb.tsx

```diff
@@ -399,6 +402,32 @@ export default class DualThumb extends React.Component<Props, State> {
     }
   }
 
+  @autobind
```

Comment:
All you want is for the "hitbox" to be larger than the "visual" line of the track, correct?

Is there no way to pull that off with CSS? I worry when I see `getBoundingClientRect` being used and the potential workload involved in calculating this stuff each time the user clicks.

I don't have much time to dig into this today unfortunately 😢 but would love to hear more about other solutions you have explored.

Reply:
The single thumb can be achieved with CSS because the native input includes the functionality of being able to click somewhere on the track and move the thumb.

The dual thumb is built manually and would need to recreate this behaviour before being able to use CSS to widen the hit area.

Reply:
sorry - is it the `thumb` or the `track` that is "too thin to click"? Which are you trying to increase the `hitbox` for?

Reply:
The track.